openmdao/recorders/recording_manager.py

```diff
@@ -0,0 +1,86 @@
+import itertools
+import time
+from openmdao.core.mpi_wrap import MPI
+
+class RecordingManager(object):
+    def __init__(self, *args, **kargs):
+        super(RecordingManager, self).__init__(*args, **kargs)
+        self._root = None 
```

Comment:
Can you remove the _root attribute and just have it passed in to the methods that need it.  I don't like the circular reference there.